cmd/hooks/wait-paths/main.go

```diff
@@ -38,14 +45,16 @@ func main() {
 		},
 	}
 	app.Action = run
-	if err := app.Run(os.Args); err != nil {
-		logrus.Fatalf("%s\n", err)
-	}
-	os.Exit(0)
+	return app
 }
 
 func run(cCtx *cli.Context) error {
 	timeout := cCtx.GlobalInt(timeoutFlag)
+
+	pathsVal := cCtx.GlobalString(pathsFlag)
```

Comment:
not really relevant, but switching to a [`StringSlice`](https://pkg.go.dev/github.com/urfave/cli/v2#Context.StringSlice) may be easier to deal with

Reply:
maybe something to improve in the future, don't really want to touch the policy enforcement code right now.